src/promptflow-core/promptflow/_constants.py

```diff
@@ -12,6 +12,7 @@
 PROMPTFLOW_SECRETS_FILE = "PROMPTFLOW_SECRETS_FILE"
 PF_NO_INTERACTIVE_LOGIN = "PF_NO_INTERACTIVE_LOGIN"
 PF_RUN_AS_BUILT_BINARY = "PF_RUN_AS_BUILT_BINARY"
+PROMPTFLOW_FLOW_INIT_CONFIG = "PROMPTFLOW_FLOW_INIT_CONFIG"
```

Comment:
Use 'PF_' prefix to align with existing env vars.

Reply:
Seems we have both PROMPTFLOW_ and PF_ ...

Reply:
Most of the 'PROMPTFLOW' are legacy env vars, and the below 'PROMPTFLOW_SECRETS_FILE' has a long history and now only used in tests. We'd better to use PF_ for the future vars.

Reply:
I saw that the other commands for Docker Run before all started with "PROMPTFLOE". If we change this command to start with "PF" separately, it looks strange.
![image](https://github.com/microsoft/promptflow/assets/23182548/19359dcc-34c7-4b30-9861-c2261f382208)

Reply:
Let's go with PF_ prefix 
See example on public environment variables like `PF_WORKER_COUNT` & `PF_LOGGING_LEVEL`
https://microsoft.github.io/promptflow/how-to-guides/faq.html#set-logging-level

Reply:
@wxpjimmy Does it sounds good to you we should align to use same pattern here? https://microsoft.github.io/promptflow/how-to-guides/deploy-a-flow/deploy-using-docker.html?highlight=promptflow_worker_num#run-with-flask-serving-engine

Reply:
ok

Reply:
done